From: Bernhard Schmidt <berni@debian.org>
Subject: Fix FTBFS with headers from Linux 4.8+
Bug-Debian: https://bugs.debian.org/844869

Linux 4.8+ adds a few includes to linux/if_tunnel.h, which conflict with
concurrent use of netinet/ip.h. Drop the latter and manually define IP_DF
which is not found anywhere else
--- a/src/tunnel.c
+++ b/src/tunnel.c
@@ -18,10 +18,13 @@
 #include <sys/ioctl.h>
 #include <sys/socket.h>
 #include <arpa/inet.h>
-#include <netinet/ip.h>
 #include <net/if.h>
 #include <linux/if_tunnel.h>
 
+#ifndef IP_DF
+	#define IP_DF	0x4000		/* dont fragment flag */
+#endif
+
 #ifdef HAVE_CONFIG_H
 	#include <config.h>
 #endif

